src/tests/Interop/Swift/README.md

```diff
@@ -0,0 +1,96 @@
+# .NET Swift Interop
```

Comment:
https://github.com/dotnet/designs/blob/main/proposed/swift-interop.md is more complete. Should this doc be deleted?

Reply:
Yes, this is not needed anymore and will be removed.

Reply:
Removed.